app/forms/declaration/Document.scala

```diff
@@ -62,7 +63,7 @@ object Document {
 
 case class PreviousDocumentsData(documents: Seq[Document])
 
-object PreviousDocumentsData {
+object PreviousDocumentsData extends DeclarationFieldCompanion {
```

Comment:
What's a `DeclarationFieldCompanion`? Seems strangely named

Reply:
This has been the name we came up with to work the `back-link` navigation.

Reply:
Renamed to `DeclarationPage`